script/netlify.sh

```diff
@@ -6,8 +6,8 @@ set -xeuo pipefail
 # https://github.com/netlify/build-image) but for now it's way simpler to just
 # accept that Netlify will need separate instructions.
 
-# get our dependencies
-npm install
+# get our dependencies (--ignore-scripts=false is needed for puppeteer)
+npm install --ignore-scripts=false
```

Comment:
why do we need puppeteer to build for Netlify? I think it's just `elm make` and `browserify`.